frontend/packages/dev-console/src/utils/__tests__/pipeline-augment.spec.ts

```diff
@@ -328,3 +296,34 @@ describe('Pipeline exists test to determine whether a pipeline is linked to a pi
     expect(pipelineFound).toBeFalsy();
   });
 });
+
+describe('Pipelinerun graph to show the executed pipeline structure', () => {
```

Comment:
We are missing a test for the actual result. You definitely have tested the paths to our current usage, but the method is, in isolation, just creating a Pipeline. We don't have a test to make sure it did do that. Goals of Unit Tests for functions should be to cover edge cases, any conditional paths, intended uses today and the function is performing the action it prescribes to - which in this case is making a PL CR out of the PLR CR.